lib/galaxy/webapps/tool_shed/api/repositories.py

```diff
@@ -81,14 +81,14 @@ def add_repository_registry_entry( self, trans, payload, **kwd ):
             response_dict[ 'message' ] = error_message
             return response_dict
         # Update the repository registry.
-        trans.app.repository_registry.add_entry( repository )
+        self.app.repository_registry.add_entry( repository )
         response_dict[ 'status' ] = 'ok'
         response_dict[ 'message' ] = 'Entries for repository %s owned by %s have been added to the Tool Shed repository registry.' \
             % ( name, owner )
         return response_dict
 
     @web.expose_api_anonymous
-    def get_ordered_installable_revisions( self, trans, name, owner, **kwd ):
+    def get_ordered_installable_revisions( self, trans, **kwd ):
```

Comment:
@davebx you can also test this with the ansible-tools role https://github.com/galaxyproject/ansible-galaxy-tools
It is not possible to install tools from the TTS anymore.